src/media/css/lib/mixins.styl

```diff
@@ -54,3 +60,17 @@ detail--wide() {
         }
     }
 }
+
+website-detail() {
+    [data-page-type~="detail"][data-page-type~="website"] & {
```

Comment:
I put it there in case we have a page that is `website` but not `detail`. Probably isn't going to happen though. I'll pull it out.